CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- Enable prometheus_exporter on EKS only, extra latency and 500 errors caused on EC2 instances running unicorn, don't expect to see the same on EKS with puma ([#231](https://github.com/alphagov/govuk_app_config/pull/231)).
```

Comment:
```suggestion
- Fix prometheus_exporter to only be enabled when the GOVUK_PROMETHEUS_EXPORTER env var is set to "true" ([#231](https://github.com/alphagov/govuk_app_config/pull/231)).
```

We should just detail what exactly the change is - the specifics about why we've done can be found in the commit message.